modules/test/playwright/tests/template-web/pages/TemplatesPage.ts

```diff
@@ -87,13 +87,22 @@ export class TemplatesPage {
 		await waitForAlert(this.page);
 	}
 
-	async createWidgetTemplate(name: string, type: string) {
+	async createWidgetTemplate(
+		name: string,
+		type: string,
+		useMoreButton = false
+	) {
 		await clickAndExpectToBeVisible({
 			autoClick: true,
 			target: this.page.getByRole('menuitem', {
 				name: type,
 			}),
-			trigger: this.page.getByRole('button', {name: 'New'}),
+			trigger: useMoreButton
+				? [
+						this.page.getByRole('button', {name: 'New'}),
+						this.page.getByRole('button', {name: 'More'}),
+					]
+				: this.page.getByRole('button', {name: 'New'}),
```

Comment:
Hey @alessiorendina I wouldn't modify existing function just for this specific use case. I would do something like this:

```typescript
const typeOption = this.page.getByRole('menuitem', {
	name: type,
});

await clickAndExpectToBeVisible({
	autoClick: true,
	target: typeOption
		.or(this.page.getByRole('button', {name: 'More'}))
		.first(),
	trigger: this.page.getByRole('button', {name: 'New'}),
});

if (typeOption.isVisible()) {
	await typeOption.click();
}
```

This way you don't event need the useMoreButton, it just check if the name exist or if the template name exists

Reply:
This way you don't need om modify clickAndExpectToBeVisible function

Reply:
`typeOption.or(this.page.getByRole('button', {name: 'More'})).first()`

This line will take the name of the template if present or the more button

Reply:
Done. Thank you, @victorg1991!